Nextest.toml

```diff
@@ -0,0 +1,20 @@
+# This is configuration for `cargo nextest`.
+
+default-profile = "diem"
+
+[profiles.diem]
+name = "diem-run"
+failure-output = "immediate"
+
+# Profile used in CI tests.
+[profiles.ci-test]
+name = "diem-ci-test-run"
+retries = 2
+# Output failures at the end of the run to make it easier to scan logs.
+failure-output = "immediate-final"
+metadata-key = "diem-junit"
```

Comment:
By default, runs don't output JUnit information, but in CI they do.

Reply:
can that be enabled with a flag to go to a standard location?

Reply:
Hmm, do you mean like a standard CLI flag which instructs that JUnit be output?

In the future I'd also like to leave open the possibility of configuring more things about the output: potentially single vs multiple files, adding extra properties to junit, adding skipped tests, etc. I think there's a lot of potential utility in having this be in a config file.

The other reason I want to do it this way is because of how GHA picks which version of an action to use. Based on what I've observed, GHA's algorithm for `pull_request` events appears to be:
* If the action file has changed, use the version in the PR.
* Otherwise use the version from the target branch.

This means that if the nextest CLI changes, most people will start seeing somewhat bogus CI errors until they rebase.